bobo.py

```diff
@@ -5,6 +5,7 @@
 from datetime import datetime
 
 from holidays import get_today_holiday
+from lessons import get_lesson_start, get_lesson_end
```

Comment:
The file is called `lesson.py` therefore you should import from `lesson`, not `lessons`. Probably a typo but I want to make sure you get it right. And the functions are in a plural form. 

In line 54 you are using `LESSON_HOURS` but it's also needed to be imported
```suggestion
from lesson import get_lessons_start, get_lessons_end, LESSON_HOURS
```